minari/dataset/minari_dataset.py

```diff
@@ -386,3 +381,11 @@ def update_dataset_from_buffer(self, buffer: List[dict]):
 
     def __iter__(self):
         return self.iterate_episodes()
+
+    def __getitem__(self, idx: int) -> EpisodeData:
```

Comment:
All changes look good, but please add  tests for `__iter__ `,`__getitem__` and `__len__`

Reply:
Much easier to add after we merge https://github.com/Farama-Foundation/Minari/pull/101